### PR TITLE
refactored test-cmd/edit to use new helper methods

### PR DESCRIPTION
**Pull request by stevekuznetsov**
/cc @deads2k @smarterclayton

### CODE REVIEW

**Comment by deads2k** [comment]
[test]

File: test/cmd/edit.sh [hunk 1]
```diff
@@ -6,15 +6,16 @@ set -o pipefail
...
 source "${OS_ROOT}/hack/util.sh"
+source "${OS_ROOT}/hack/cmd_util.sh"
 os::log::install_errexit
 
 # This test validates the edit command
 
-oc create -f examples/hello-openshift/hello-pod.json
+os::cmd::expect_success 'oc create -f examples/hello-openshift/hello-pod.json'
 
-[ "$(OC_EDITOR=cat oc edit pod/hello-openshift 2>&1 | grep 'Edit cancelled')" ]
```
Comment (stevekuznetsov) [inline test/cmd/edit.sh]: `2>&1` is not necessary any longer, as it was previously used for the `grep` and the helper methods implicitly will check both `stdout` and `stderr` for the desired text.

**Comment by stevekuznetsov** [comment]
I've never seen this before... 
```
!!! Error in test/cmd/../../hack/text.sh:53
	'tput setaf 2' exited with status 1
```
@liggitt this command is used in `test-integration`, right? Do you know why this would fail?

**Comment by stevekuznetsov** [comment]
re[test]

**Comment by stevekuznetsov** [comment]
Changed `hack/text.sh` to use functions instead of aliases, added a TTY-check. If this works I'll break that commit out into a separate pull.
[test]

**Comment by stevekuznetsov** [comment]
Tagging @kargakis for review - please review that we are expecting success and/or failure correctly, as the exit codes were masked previously with the `grep` afterword.

**Comment by deads2k** [comment]
lgtm [merge]

**Pull request merged**